Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -757,4 +753,24 @@ public static String checkPrefixBeforeAppend(final String checkValue, final Stri
             return checkValue;
         }
     }
+
+    public static List<Object> checkHcidPrefixInList(final List<Object> valueList){
+        List<Object> tempList = new ArrayList<>();
+        for(Object eachValue : valueList){
+            tempList.add(checkPrefixBeforeAppend(eachValue.toString(), NhincConstants.HCID_PREFIX));
+        }
+        return tempList;
+    }
+
+    protected boolean isAcpOrIacpExists(final CallbackProperties properties) {
+        final List accessConstentValues = properties.getEvidenceAccessConstent();
+        final List evidenceInstanceAccessConsentValues = properties.getEvidenceInstantAccessConsent();
+
+        //return false if ACP/IACP does not exist
+        //only created the AuthzDecisionStatement if true
+        if(CollectionUtils.isEmpty(accessConstentValues) && CollectionUtils.isEmpty(evidenceInstanceAccessConsentValues)){
```

Comment:
This should have popped up on sonarqube.  Can be simplified to:

return !Collections.IsEmpty(accessConsentValues) && !CollectionUtils.isEmpty(evidenceInstanceAccessConsentValues);

No need for if statement.